onelive/archive/functions.js

```diff
@@ -2,7 +2,7 @@ let nextPageToken = null;
 let fetchedItemCount = 0;
 
 function loadYoutubeVideos() {
-    let url = "https://apps.sefglobal.org/youtube/onelive?maxResults=12";
+    let url = "https://dataholder.sefglobal.org/youtube/onelive?maxResults=3";
```

Comment:
```suggestion
    let url = "https://dataholder.sefglobal.org/youtube/onelive?maxResults=12";
```

@gpsakthivel, here maxResults count should be 12

Reply:
Resolved